app/graph/mutations/user_disconnect_login_account_mutation.rb

```diff
@@ -0,0 +1,21 @@
+UserDisconnectLoginAccountMutation = GraphQL::Relay::Mutation.define do
+  name 'UserDisconnectLoginAccount'
+
+  input_field :id, !types.Int
+
+  input_field :provider, !types.String
+
+  return_field :success, types.Boolean
+
+  return_field :user, UserType
+
+  resolve -> (_root, inputs, _ctx) {
+    user = User.where(id: inputs[:id]).last
```

Comment:
What prevents a user from disconnecting an account of another user? I think we should load from `User.current`... can we do that?

Reply:
Sure will depend on `User.current`